tests/kafkatest/services/connect.py

```diff
@@ -140,6 +141,12 @@ def pause_connector(self, name, node=None):
     def resume_connector(self, name, node=None):
         return self._rest('/connectors/' + name + '/resume', method="PUT")
 
+    def list_connector_plugins(self, node=None, retries=0, retry_backoff=.01):
+        return self._rest_with_retry('/connector-plugins/', node=node, retries=retries, retry_backoff=retry_backoff)
```

Comment:
This should never hit a 409, so `_rest_with_retry` shouldn't be necessary as far as I can tell.